docs/tool_jstat.md

```diff
@@ -0,0 +1,59 @@
+﻿<!--
+* Copyright (c) 2019, 2019 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Java statistics monitoring (`jstat`) tool
+
+Use the `jstat` tool to obtain Java Virtual Machine (JVM) statistics. The tool is similar to the HotSpot tool of the same name; the OpenJ9 version of `jstat` is an independent implementation, added for compatibility.
+
+The command syntax is as follows:
+
+    jstat [<option>] [<vmid>]
+
+Where `vmid` is the Attach API virtual machine identifier for the Java process. This ID is typically the same as
+the operating system *process ID*, unless you specified the `-Dcom.ibm.tools.attach.id` system property when you started the process.
```

Comment:
Would be good to cross reference:  [`-Dcom.ibm.tools.attach.id`](dcomibmtoolsattachid.md)

Reply:
fixed.